packages/site/src/lib/components/modals/Contact.svelte

```diff
@@ -89,7 +108,7 @@
         bind:value={message}
         class="form-input bg-white w-full"
         placeholder={$t('contact.enter_message', {
-          default: 'Enter your message',
+          default: 'Please write your message in English, French or Spanish',
```

Comment:
Why? We can use Google Translate if they use another language. If they know English or Spanish they will most likely use that. If they don't then another is fine.